CHANGELOG.md

```diff
@@ -1,6 +1,28 @@
 v1.x
 ===================
 
+* 1.5.0
+  * [BC BREAK] change default connection timeout from 10s to 4s, request timeout from 10s to 8s
+  * [BC BREAK] change bundle configuration:
```

Comment:
This pretty much constitutes version 2.0. If you don't want to bump major just need to convert old configuration to new format (under name `default` or such) automatically

Reply:
Reasonable notice, thanks,. Changed to 2.0.0